### PR TITLE
Update getting started doc with simple hello world tutorial

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

**Comment by danroth27** [comment]
@Rick-Anderson @ardalis @rachelappel @DamianEdwards Please review

**Comment by danroth27** [comment]
I'll make these final changes as part of https://github.com/aspnet/Docs/pull/1220.

File: aspnet/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    :language: c#
+    :emphasize-lines: 2,10-15
+
+7. Build and run the app:
+
+  .. code-block:: console
+  
+    dotnet run
+
+8. Browse to \http://localhost:5000 to see the running output:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: Can we just leave it as 
Browse to \http://localhost:5000 
It's not running output and they know what to expect.

File: aspnet/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    :language: c#
+    :emphasize-lines: 11
+
+4. Restore the packages for your project:
+
+  .. code-block:: console
+    
+    dotnet restore
+
+5. Add a ``Startup.cs`` file that defines the request handling logic your app:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: logic **FOR** your app:
My pref would be:
Add a ``Startup.cs`` file that defines the request handling logic:

File: aspnet/getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Create a new .NET Core project:
+
+  .. code-block:: console
+    
+    mkdir aspnetcoreapp
+    cd aspnetcoreapp
+    dotnet new
+
+3. Update your ``project.json`` file to add the package for the Kestrel HTTP server and the ASP.NET Core hosting model:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: *project.json* is italic, not code.
I prefer to refer to **THE** file, The project in lieu of YOUR file, YOUR project, etc

File: aspnet/getting-started.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/project.json
+    :language: c#
+    :emphasize-lines: 11
+
+4. Restore the packages for your project:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: Restore the packages:

File: aspnet/getting-started.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  .. code-block:: console
+    
+    dotnet restore
+
+5. Add a ``Startup.cs`` file that defines the request handling logic your app:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/Startup.cs
+    :language: c#
+
+6. Replace the code in the static ``Main`` method in ``Program.cs`` to setup and start the Web host:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: Files are *italic* not code - *Program.cs*

File: aspnet/getting-started.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: Are you 100% sure https://microsoft.com/net/core will work? you could use a fwlink. I'm guessing we'll want to update this to a dedicated install page.

File: aspnet/getting-started.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/Startup.cs
+    :language: c#
+
+6. Replace the code in the static ``Main`` method in ``Program.cs`` to setup and start the Web host:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/Program.cs
+    :language: c#
+    :emphasize-lines: 2,10-15
+
+7. Build and run the app:
```
Comment (Rick-Anderson) [inline aspnet/getting-started.rst]: Probably leave as is - but consider:
Run the app (the ``dotnet run`` command will build the app when it's out of date):

File: aspnet/getting-started.rst (hunk 3 above)
Comment (danroth27) [inline aspnet/getting-started.rst]: You actually only add the Kestrel package as a direct dependency, which then pulls in a bunch of stuff. I'll simplify the text.

File: aspnet/getting-started.rst (hunk 6 above)
Comment (danroth27) [inline aspnet/getting-started.rst]: I'm 100% sure.

**Pull request merged**